pkgs/by-name/at/atlauncher/package.nix

```
@@ -1,4 +1,16 @@
-{ copyDesktopItems, fetchurl, jre, lib, makeDesktopItem, makeWrapper, stdenv, udev, xorg }:
+{ copyDesktopItems
+, fetchurl
+, jre
+, lib
+, makeDesktopItem
+, makeWrapper
+, stdenv
+
+, # dependencies
+  libglvnd
+, udev
+, xorg
+}:
```

```suggestion

, gamemodeSupport ? stdenv.isLinux
, textToSpeechSupport ? stdenv.isLinux
, additionalLibs ? [ ]
}:

let
  runtimeLibs = [
    libglvnd
    libpulseaudio
    udev
    xorg.libXxf86vm
  ]
  ++ lib.optional gamemodeSupport gamemode.lib
  ++ lib.optional textToSpeechSupport flite
  ++ additionalLibs;
in

```

Taking inspiration from the [Prismlauncher wrapper](https://github.com/NixOS/nixpkgs/blob/1863d9cfeb0a36da5ff3a2896eb7ed9c7a7d6f77/pkgs/games/prismlauncher/wrapper.nix#L91), we can extract the necessary libs into one single `runtimeLibs`. Also, by adding an `additionalLibs` list, we can easily add new libraries without having to override the entire wrapper.